app/Utils/TabulatorPaginator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Utils;
+
+use Illuminate\Pagination\LengthAwarePaginator;
+
+class TabulatorPaginator
```

Comment:
Isn't it possible to add translations for the paginator? For next, last, etc.

Reply:
It is, will open an issue for this after merge.